### ISSUE TITLE
 MoneroOcean- xmrig won't connect to Monero p2pool

### ISSUE THREAD

**Issue by Santahr** [MoneroOcean/xmrig#104]
**Describe the bug**
127.0.0.1 read error: "connection reset by peer"

**To Reproduce**
xmrig.exe -o 127.0.0.1
or xmrig.exe -o address of p2pool

**Expected behavior**
Vanilla xmrig works!

**Comment by Spudz76** [comment]
Assuming p2pool is intolerant of extra junk in the login packet (`algo-perf`) when it should be simply ignoring unknown extensions.

**Comment by Spudz76** [comment]
Nope, looks like it does ignore extra junk, so perhaps you've left `"tls": true` on the pool definition.

Note that config.json always wins over command line options.

**Comment by Santahr** [comment]
** To Reproduce **
    "pools": [
        {
            "algo": null,
            "coin": null,
            "url": "gulf.moneroocean.stream:20128",
            "user": "wallet add",
            "pass": "x",
            "rig-id": "name",
            "nicehash": false,
            "keepalive": true,
            "enabled": **false**,
            "tls": true,
            "tls-fingerprint": null,
            "daemon": false,
            "socks5": null,
            "self-select": null,
            "submit-to-origin": false
        },
{
            "coin": "monero",
            "algo": "rx/0",
            "url": "127.0.0.1:3333",
	    "enabled": **true**,
            "tls": false,
            "keepalive": true,
            "nicehash": false
        }
    ],

**END Results :** 

read error: "connection reset by peer"
